#2190 Fixed feature 'Apply new properties basing on' for property 'Event text renderer properties'

Patrykb0802 opened the pull request:
- Added eventTextRenderer as argument in jQuery().val()

Patrykb0802 committed:
#2190 Fixed feature 'Apply new properties basing on' for property 'Event text renderer properties'

- Added eventTextRenderer as argument in jQuery().val()

Limraj reviewed (commented):
1. **Work for binary and numeric.** 
**For mulstistate, it sets the event text type of the renderer, but does not transfer the value:**
![Screenshot 2023-08-09 at 18 55 50](https://github.com/SCADA-LTS/Scada-LTS/assets/35842300/0ab4da80-9dd5-412e-a8bd-7f0c95266b04)
After apply:
![Screenshot 2023-08-09 at 18 55 26](https://github.com/SCADA-LTS/Scada-LTS/assets/35842300/891804da-681e-4a50-8463-e243ba5de713)
**There is no Event Text Renderer type for the alphanumeric type - ok.**

3. There is no information about the Event Text Renderer in the information window, what will be transferred:
![Screenshot 2023-08-09 at 18 51 03](https://github.com/SCADA-LTS/Scada-LTS/assets/35842300/38ea3061-ba9a-42f7-bb28-199063939709)

Patrykb0802 committed:
#2190 Fixed feature 'Apply new properties basing on' for property 'Event text renderer properties'

- Added keywords in some variables to fix applying and displaying changes of event text renderer

Limraj reviewed (approved):
Apply function 'Event text renderer properties' from one point to another works for data types:
- numeric;
- binary;
- multistate;
function is not supported for alphanumeric type.
